### PR TITLE
Konnector Electrabel

### PR DESCRIPTION
**Pull request by ZeHiro**
Konnector to import Electrabel bills (Belgian energy provider)

### CODE REVIEW

File: server/konnectors/electrabel.coffee [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+                notifContent = null
+                if entries?.filtered?.length > 0
+                    localizationKey = 'notification electrabel'
+                    options = smart_count: entries.filtered.length
+                    notifContent = localization.t localizationKey, options
+
+                callback err, notifContent
+
+# Procedure to login to Free website.
```
Comment (frankrousseau) [inline server/konnectors/electrabel.coffee]: Could you update the comment?

Comment (frankrousseau) [inline server/konnectors/electrabel.coffee]: Don't worry I'll fix it.

**Comment by frankrousseau** [comment]
Wow thank you @ZeHiro!

**Comment by ZeHiro** [comment]
Living both in France and Belgium helps to have a lot of providers :D

**Pull request merged**